carma_wm/src/IndexedDistanceMap.h

```diff
@@ -0,0 +1,195 @@
+#pragma once
```

Comment:
This file looks like it is a duplicate. This was moved to include in a previous PR.